data/data/azure/main.tf

```diff
@@ -170,8 +170,8 @@ resource "azurerm_storage_blob" "rhcos_image" {
   storage_account_name   = azurerm_storage_account.cluster.name
   storage_container_name = azurerm_storage_container.vhd.name
   type                   = "Page"
```

Comment:
Page is ok here?

Reply:
Yes. But only for VHD files.

My initial commit was overwritten by someone force pushing to the installer repo. 

Vadim restored my commit partially. Because of that "Page" is already in the fcos branch. But source_uri is missing.

In addition someone was setting Page also to the bootstrap.ign which does require proper byte alignment that bootstrap.ign does not have, throwing an error. And also this is not required.

I know: confusing.